### PR TITLE
Context Menu / Menu: additional changes in data and typo's

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: NetCoreDemos/Views/Menu/Scrolling.cshtml [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <div class="option">
             @(Html.DevExtreme().CheckBox()
                 .Value(false)
-                .Text("Limit submenus height by 200 px")
+                .Text("Limit submenus height to 200 px")
```
Comment (EugeniyKiyashko) [inline NetCoreDemos/Views/Menu/Scrolling.cshtml]:
```suggestion
                .Text("Limit submenu height to 200 px")
```

File: MVCDemos/Models/SampleData/ContextMenuData.cs [hunk 2]
```diff
@@ -42,152 +42,152 @@ public static class ContextMenuData {
...
                                 text = "Senior Cell Phones"
                             },
-                            new ContextMenuScrollingItem { 
+                            new ContextMenuScrollingItem {
                                 text = "Mobile Broadband Devices"
                             },
-                            new ContextMenuScrollingItem { 
-                                text = "Cell phones Accessories"
+                            new ContextMenuScrollingItem {
+                                text = "Cell Phones Accessories"
```
Comment (EugeniyKiyashko) [inline MVCDemos/Models/SampleData/ContextMenuData.cs]:
```suggestion
                                text = "Cell Phone Accessories"
```

File: MVCDemos/Models/SampleData/MenuScrollingData.cs [hunk 3]
```diff
@@ -266,7 +266,7 @@ public static class MenuScrollingData {
                 {
                     new MenuItem
                     {
-                        text = "Sport & Outdoors",
+                        text = "SportS & Outdoors",
```
Comment (EugeniyKiyashko) [inline MVCDemos/Models/SampleData/MenuScrollingData.cs]:
```suggestion
                        text = "Sports & Outdoors",
```

Comment (nikkithelegendarypokemonster) [reply MVCDemos/Models/SampleData/MenuScrollingData.cs]: [9346045](https://github.com/DevExpress/devextreme-demos/pull/3216/commits/9346045f991302d17d6e308fb5feea02a42aa01d)

File: MVCDemos/Models/SampleData/ContextMenuData.cs (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Models/SampleData/ContextMenuData.cs]: [90992d5](https://github.com/DevExpress/devextreme-demos/pull/3216/commits/90992d5d403669cf0225191ae41495528f84c1e4)

File: NetCoreDemos/Views/Menu/Scrolling.cshtml (hunk 1 above)
Comment (nikkithelegendarypokemonster) [reply NetCoreDemos/Views/Menu/Scrolling.cshtml]: [90992d5](https://github.com/DevExpress/devextreme-demos/pull/3216/commits/90992d5d403669cf0225191ae41495528f84c1e4)

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Pull request merged**